docs/guides/social-signin/get-tokens.mdx

```diff
@@ -0,0 +1,126 @@
+---
+id: get-tokens
+title: Get Social Sign-in Provider Tokens
+---
+
+You can get the OIDC / OAuth 2.0 access, refresh, and ID tokens issued for the Identity by social sign-in providers. 
+
+These tokens are issued only when the Identity:
+- Signs up with a social sign-in provider. 
+- Links a new social sign-in provider to their account. 
+
+Run this command to get the Identity details that include the social sign-in provider tokens: 
+
+````mdx-code-block
+import Tabs from '@theme/Tabs'
+import TabItem from '@theme/TabItem'
+
+<Tabs>
+<TabItem value="Ory Cloud">
+
+Use the Ory CLI: 
+
+```shell
+ory get identity <IDENTITY_ID> --project <ORY_CLOUD_PROJECT_ID> -i oidc --format yaml
```

Comment:
Does this actually work? :D Have you tried it?

Reply:
Yes, I have tried it and yes, it does work.

Reply:
👍